src/index.html

```diff
@@ -84,23 +85,264 @@ <h2 class="hero__social-media-links__title">Let&apos;s connect</h2>
 
             <!-- Other -->
             <li>
-              <a href="#"
-                ><!-- TODO: What social media has this icon? -->
+              <a href="#">
+                <!-- TODO: What social media has this icon? -->
                 <img src="assets/icons/unknown.svg" alt="Social media icon" />
               </a>
             </li>
           </ul>
         </div>
       </section>
 
-      <!-- Project Cards -->
-      <section class="container">
-        <!-- Social Media Links -->
+      <!-- Works Section -->
+      <section class="works container">
+        <ul class="works__cards-list">
+          <!-- Project-1 Card -->
+          <li class="works__project-card">
+            <!-- Project Screenshot -->
+            <img
+              class="works__project-card__screenshot"
+              src="assets/project-screenshots/proj-1.png"
+              alt="Screenshot of Tonic project"
+            />
+
```

Comment:
I suggest removing the white spaces in your code, by following this your project will look more. professional. 😄